tests/worksheets/src/test/scala/tests/worksheets/WorksheetSuite.scala

```diff
@@ -158,7 +158,7 @@ class WorksheetSuite extends BaseSuite {
       |val n = User("Susan")
       |""".stripMargin,
     """|case class User(name: String)
-       |<val n = User("Susan")> // User("Susan")
+       |<val n = User("Susan")> // : User = User("Susan...
```

Comment:
Would it make sense to special case this scenario and drop the type when the value starts with the same syntax as the type? Feel free to ignore, just an idea

Reply:
I think it's better to be consistent, otherwise it might be surprising to users. And it might be quite fragile I think.

It would need to be something like:
```scala
            if (isSingle && !chunk.startsWith(binder.tpeString + "("))
              out
                .append(": ")
                .append(binder.tpeString)
                .append(" = ")
            out.append(chunk)
```